src/testFixtures/java/uk/gov/di/ipv/cri/common/library/client/CommonApiClient.java

```diff
@@ -1,17 +1,34 @@
 package uk.gov.di.ipv.cri.common.library.client;
 
+import software.amazon.awssdk.auth.credentials.AwsSessionCredentials;
+import software.amazon.awssdk.http.SdkHttpFullRequest;
+import software.amazon.awssdk.http.SdkHttpMethod;
+import software.amazon.awssdk.http.SdkHttpRequest;
+import software.amazon.awssdk.http.auth.aws.signer.AwsV4HttpSigner;
+import software.amazon.awssdk.http.auth.spi.signer.SignedRequest;
+import software.amazon.awssdk.regions.Region;
 import uk.gov.di.ipv.cri.common.library.util.URIBuilder;
 
 import java.io.IOException;
+import java.net.URI;
 import java.net.http.HttpClient;
 import java.net.http.HttpRequest;
 import java.net.http.HttpResponse;
+import java.util.List;
+import java.util.Map;
 
 public class CommonApiClient {
-
     private final HttpClient httpClient;
     private final ClientConfigurationService clientConfigurationService;
 
+    private static final String STACK_NAME = "test-resources";
```

Comment:
Can we make STACK_NAME and env var too, please, so we can test using different stacks. The default value should be `test-resources` as is but it should be possible to override it if needed